bluesky/plans.py

```diff
@@ -17,7 +17,7 @@
 
 from . import utils
 from .utils import Msg
-
+import math
```

Comment:
Might as well use `numpy.log` here (and in the tests) for consistency and to avoid paying the overload of importing `math`.

Reply:
@danielballan `numnpy.log` does not have the 'base' kwarg that is required here, hence why i used `math.log` instead.